packages/common/src/authentication/AuthContext.tsx

```diff
@@ -124,20 +121,27 @@ export const AuthProvider: FC = ({ children }) => {
 
   // returns MRU store, if set
   // or the first store in the list
-  // TODO: return default store rather than first - currently not provided by API
-  const getStore = async () => {
-    if (mostRecentlyUsedCredentials?.store)
+  const getStore = async (token?: string) => {
+    const userDetails = await getStores(token);
+    const defaultStore = userDetails?.defaultStore;
+    const stores = userDetails?.stores.nodes;
+
+    if (
+      mostRecentlyUsedCredentials?.store &&
+      stores?.some(store => store.id === mostRecentlyUsedCredentials?.store?.id)
+    ) {
       return mostRecentlyUsedCredentials.store;
+    }
 
-    const { nodes } = await getStores();
-    return !!nodes && nodes?.length && nodes?.length > 0
-      ? nodes?.[0]
-      : undefined;
+    if (!!defaultStore) return defaultStore;
```

Comment:
default store is now available! as a separate property though, rather than a prop on a store.. so I have to do it this way